app.js

```diff
@@ -51,15 +50,17 @@ async function findCountries(name, continent) {
 searchForm.addEventListener('submit', (e) => {
     e.preventDefault();
     const formData = new FormData(searchForm);
-    // > Part C: Call findCountries with name and continent from formData
+    findCountries(formData.get('name'), formData.get('continent'));
```

Comment:
this totally works, you can also store the name and continent in variables to increase readability